components/motor/motor.go

```diff
@@ -97,8 +97,8 @@ type Motor interface {
 	// revolutions at a given speed in revolutions per minute. Both the RPM and the revolutions
 	// can be assigned negative values to move in a backwards direction. Note: if both are
 	// negative the motor will spin in the forward direction.
-	// If revolutions is 0, this will run the motor at rpm indefinitely
 	// If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
+	// Deprecated: If revolutions is 0, this will run the motor at rpm indefinitely.
```

Comment:
just curious are we going to get rid of this behavior entirely once setRPM is done?